mlflow/cli.py

```diff
@@ -73,7 +73,7 @@ def cli():
 def run(uri, entry_point, version, param_list, experiment_id, mode, cluster_spec, git_username,
         git_password, no_conda, new_dir, storage_dir):
     """
-    Run an MLflow project from the given URI.
+    Run an MLflow project from the given URI, blocking till the run completes.
```

Comment:
This might make the description too long to show up as a short description in `mlflow --help`. Try it out. If it's too long, explain the blocking farther below.